doc/platforms/x86_64_linux/x86_64_android.md

```diff
@@ -0,0 +1,113 @@
+# Edge Orchestration on x86_64 (TypeExec: android)
+
+## Prerequisites:
+- Android SDK
+1. Download [Android SDK](https://developer.android.com/studio/index.html#command-tools), then extract the downloaded package.
+2. Wrapping `tools` directory inside `cmdline-tools`
+3. Execute folowing commands: 
+```
+$ ./sdkmanager "platform-tools" "platforms;android-28"
+$ export ANDROID_SDK=<path to Android SDK directory>
+$ export ANDROID_HOME=<path to Android SDK directory>
+$ export PATH=$PATH:$ANDROID_SDK/cmdline-tools/tools:$ANDROID_SDK/platform-tools
+```
+- Android NDK
+1. Download the latest version of [Android NDK](https://developer.android.com/ndk/downloads), then extract the downloaded file.
+2. Execute folowing commands:
```

Comment:
I don't know why your guide didn't work for me but I made it as below.
1. Go to cmdline-tools/tools/bin and Execute following commands:
```
$ ./sdkmanager --install ndk-bundle
```

Reply:
Done. Thank you!